From b67a4befac19e46f48e7955b5475dca89e8c9d52 Mon Sep 17 00:00:00 2001
From: Mehdi Sabwat <mehdi@videolabs.io>
Date: Fri, 16 Apr 2021 11:32:33 +0200
Subject: [PATCH 1/1] modules: disable libvlc_json and ytbdl vlc.js#17

The libjson library is not linkable with emsdk v2.0.17,
this commit should be reverted when the ticket is resolved.
---
 modules/demux/Makefile.am | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/modules/demux/Makefile.am b/modules/demux/Makefile.am
index 48f5d7a97d..dbfe923650 100644
--- a/modules/demux/Makefile.am
+++ b/modules/demux/Makefile.am
@@ -529,9 +529,11 @@ libytdl_plugin_la_SOURCES = demux/ytdl.c
 libytdl_plugin_la_LIBADD = libvlc_json.la
 if !HAVE_WIN32
 if !HAVE_ANDROID
+if !HAVE_EMSCRIPTEN
 demux_LTLIBRARIES += libytdl_plugin.la
 endif
 endif
+endif
 
 libnoseek_plugin_la_SOURCES = demux/filter/noseek.c
 demux_LTLIBRARIES += libnoseek_plugin.la
@@ -549,4 +551,6 @@ libvlc_json_la_SOURCES = \
 libvlc_json_la_CPPFLAGS = $(AM_CPPFLAGS) -I$(srcdir)/demux/json
 libvlc_json_la_LIBADD = $(LTLIBVLCCORE) ../compat/libcompat.la $(LIBM)
 libvlc_json_la_LDFLAGS = -static
+if !HAVE_EMSCRIPTEN
 noinst_LTLIBRARIES += libvlc_json.la
+endif
-- 
2.32.0

